src/components/live/home.tsx

```diff
@@ -1,5 +1,122 @@
+import { api } from "~/utils/api";
+import { Button } from "../ui/button";
+import { ArrowUpRight, LucideIcon } from "lucide-react";
+import Image from "next/image";
 const Home = () => {
-  return <div>home</div>;
+  const { data: application } = api.application.get.useQuery();
+
+  return (
+    <div className="flex w-full flex-row justify-start p-10 ">
+      <div className="flex w-2/3 flex-col ">
+        <h2 className="font-MagicRetro text-5xl">
+          Hi {application?.firstName ? `${application.firstName}` : "Hacker"},
+        </h2>
+        <p className="pt-5 text-lg text-slate-600">
+          Welcome to Hack Western 11, where ideas take flight!
+        </p>
+        <p className="text-md pt-5  text-slate-600">
+          We are thrilled to have you at Hack Western 11! We hope you build a
+          cool project and kindle some fun memories this weekend 💜
+        </p>
+        <div className="md:hidden">
+          <QuickLinks />
+        </div>
+        <div className="pt-5">
+          <h3 className="text-md font-bold text-slate-700">GETTING STARTED</h3>
+          <div className="w-full border-t-4 border-primary-300" />
```

Comment:
we should have a smaller border for dis

Reply:
or we can use the Separator component for this and just change the border colour